core/WFCORE-1427_ Convenient_declaration_of_server_suspend_timeout.adoc

```diff
@@ -0,0 +1,235 @@
+= Title
```

Comment:
btw - it looks like there is an unintentional space in the name of this document - `WFCORE-1427_ Convenient...`

Reply:
Fixed, thanks